sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,16 +7,13 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.39.0-preview</Version>
+		<Version>1.40.0-preview</Version>
```

Comment:
Except for csproj, please also update version in AssemblyInfo.cs

Reply:
Updated, please review